File: my-folder/0009-palindrome-number/solution.java
class Solution {
    public boolean isPalindrome(int x) {
        if (x < 0) {
            return false;
        }

        int reverse = 0;
        int origin = x;

        while (origin != 0) {
            reverse = origin % 10 + reverse * 10;
            origin = origin / 10;
        }

        return reverse == x;
    }
}

File: my-folder/0009-palindrome-number/solution.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(x) {
    if (x < 0) return false;

    let reverse = 0;
    let origin = x;

    while (origin) {
        reverse = origin % 10 + reverse * 10;
        origin = Math.floor(origin / 10);
    }

    return reverse === x;
};
